news_update.php

```diff
@@ -69,17 +69,15 @@
 
 echo '<div class="space-20"></div>';
 echo '<div class="alert alert-success">';
-echo lang_get( 'operation_successful' ) ?>
-
-echo '<br />';
+echo lang_get( 'operation_successful' );
+echo '<br /></div>';
 
 print_button( 'news_edit_page.php?news_id=' . $f_news_id . '&action=edit', lang_get( 'edit_link' ) );
+echo '&nbsp;&nbsp;&nbsp;&nbsp;';
```

Comment:
Roland, Badfiles - I've been in discussion with the FSF Licensing help desk for the past week regarding the addition of this template. I've sent the following email to the mantisbt-dev list:

Hello,

I have been in discussion with the GPL/FSF licensing team for the last week regarding a recent commit to mantisbt.

The following files were added to the open source mantisbt project which is GPL licensed:

1) https://github.com/mantisbt/mantisbt/blob/master/js/ace-elements.min.js
2) https://github.com/mantisbt/mantisbt/blob/master/js/ace-extra.min.js
3) https://github.com/mantisbt/mantisbt/blob/master/js/ace.min.js
4) https://github.com/mantisbt/mantisbt/blob/master/css/ace.min.css
5) https://github.com/mantisbt/mantisbt/blob/master/css/ace-ie.min.css
6) https://github.com/mantisbt/mantisbt/blob/master/css/ace-rtl.min.css

These files are licensed under a commercial license ( https://wrapbootstrap.com/theme/ace-responsive-admin-template-WB0B30DGR ) for which a fee may be payable by the end user.

The license options are available at http://support.wrapbootstrap.com/knowledge_base/topics/usage-licenses. 

Item 6 of the licenses state that "If the item was created using materials which are the subject of a GNU General Public License (GPL), your use of the item is subject to the terms of the GPL in place of the foregoing conditions (to the extent the GPL applies)."

A copy of the GPL license is available at https://www.gnu.org/licenses/gpl-3.0.en.html.

My understanding and suggested by the licensing@fsf.org team (although they are not lawyers) is that would be a violation of the GPL to add these files in this way.

The advice received from their team in instances like this was to have a look at http://www.gnu.org/licenses/gpl-violation.html, and reach out to the GPL project, and try to resolve the issue within the community. If that fails, their advice is to contact http://softwarefreedom.org/ who may offer free legal services to open developers.

My reason for a number of years for contributing to the MantisBT project was it was an open source project, and the original authors when i joined had said would not make money from the project. From a personal point of view, I did not want to spend my time to aid a commercial company, so that was an important goal.

Whilst I've been doing some research into the situation, badfiles asked the question of https://github.com/mantisbt/mantisbt/pull/795#discussion_r67747346 . The reply from syncguru seems to confirm that the files listed in lines 1-6 above, which I believed were from the commercial licensed ace admin template, are indeed from the commercial template, as indicated at https://github.com/mantisbt/mantisbt/pull/795#discussion_r67768649

He states:
"1- Mantishub has extended license
2- Admin themes are very good fit for bug-tracker app. Ace was top seller back then when this effort started (around May 2014) 
3- We have full source code control. Also the author is reachable on twitter."

This would seem to confirm that the code is indeed commercial, and equally that the license is owned by a separate commercial company "Mantishub". 

My questions to the FSF and their responses suggested that the first step is to reach out in a friendly manner and explain the problem - hopefully the above achieve this.

They then suggest proposing a solution, and that if that fails the next step should be contacting the Software Freedom Law Center, which provides pro-bono legal services to developers of Free, Libre, and Open Source Software (http://softwarefreedom.org/).

I'd like to firstly provide the option for the developers of the mantisbt project to provide an explanation if I've misunderstood anything in the GPL in relation to this, but having asked various linux licensing groups/people over the last week, I suspect this is unlikely.

In terms of workable solutions, I see two options:

a) The files listed in 1-6 above (and there are a couple of extra files which i've not yet evaluated) are removed from the mantisbt project. 

b) Any contributions that I (and any other contributor that feels the same) have made to the mantisbt project are removed, such that they no longer have any code in the project.

At this stage, I'm going to await other responses, and do nothing else apart from send this email and reply as appropriately.

I've been informed of the following: "However if you and others have GPL'd contributions in the project then you have the legal power to enforce the terms of the software and refuse to let anyone distribute your code linked to proprietary/non-free software."

On that basis, and in case users read this and are concerned - I'd like to make clear that at this point, that the released versions of mantisBT only contain GPL compatible code, and therefore there is no issues with distribution of that code. The code i'm emailing about only exists in the github repository.

Thanks
Paul